containers/unit_tests/TestErrorReporter.hpp

```diff
@@ -195,6 +195,7 @@ struct ErrorReporterDriverUseLambda
   void execute(int reporter_capacity, int test_size) {
     Kokkos::parallel_for(
         Kokkos::RangePolicy<execution_space>(0, test_size),
+        // NOLINTNEXTLINE(kokkos-implicit-this-capture)
```

Comment:
Wait what?  Does that mean there is a bug in the clang-tidy check?

Reply:
Yes, see https://kokkosteam.slack.com/archives/G5CBLMFLP/p1659012380757519.